src/test/resources/Suite.xml

```diff
@@ -3,7 +3,7 @@
 <suite name="Suite" verbose="1" >
     <test name="Nopackage" >
         <classes>
-            <class name="tests.CreateCampaign" />
```

Comment:
I think we should not remove implementation like this, you should just add your test and for your local test you can comment those classes.